docs/src/guide/discretisation.jl

```diff
@@ -118,7 +118,8 @@ plot(real(ψ); label="")
 
 # Again this should match with the result above.
 #
-# **Exercise 4:** Look at the Fourier coefficients of $\psi$ ($\psi$_fourier)
+# !!! "Exercise 4"
```

Comment:
I think the syntax here and the next line is not correct (some keyword missing after !!! & indention in next lines

Reply:
Right thanks !